packages/payload/src/config/types.ts

```diff
@@ -797,6 +797,14 @@ export type Config = {
      * Note: If you've modified the default paths via `admin.routes`, delete those directories instead.
      */
     disable?: boolean
+    experimental?: {
+      /**
+       * This removes the clientConfig.i18n property from the client config, reducing the amount of data sent to the client.
+       *
+       * This will be enabled by default in 4.0, as the `useLocalization` hook will be the recommended way to access i18n data.
```

Comment:
You should include an example of the `useLocalization` hook here to completely eliminate any friction from on-ramping to this suggested pattern. Might also want to throw a `@todo` flag on this so its easier for us to find/replace when needed.